src/main/java/trade/wayruha/mexc/constant/ApiEndpoints.java

```diff
@@ -0,0 +1,66 @@
+package trade.wayruha.mexc.constant;
+
+public class ApiEndpoints {
```

Comment:
Should we call it something like WSSubscriptions or so?

Reply:
I thoght that we could put heree all our API routes and use this constants. 

Like take the path from here 
 @GET("api/v3/exchangeInfo")
 trade.wayruha.mexc.service.endpoint.MarketDataAPI#getExchangeInfo